django_app/announcements/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```

Comment:
Not sure if we need this, probably with one simple test as a starting point and not completely empty

Reply:
yea, these are generated automatically when you create an app I think